Patch by Robert Scheck <robert@fedoraproject.org> for Crypt::GPG <= 1.63, which
sets the default path to the GnuPG binary equivalent to where GnuPG is installed
at Fedora and Red Hat Enterprise Linux.

--- Crypt-GPG-1.63/GPG.pm			2007-04-02 15:39:39.000000000 +0200
+++ Crypt-GPG-1.63/GPG.pm.fedora		2009-05-29 14:11:17.000000000 +0200
@@ -26,7 +26,7 @@
 ( $VERSION ) = '$Revision: 1.63 $' =~ /\s+([\d\.]+)/;
 
 sub new {
-  bless { GPGBIN         =>   '/usr/local/bin/gpg',
+  bless { GPGBIN         =>   '/usr/bin/gpg',
 	  FORCEDOPTS     =>   '--no-secmem-warning',
 	  GPGOPTS        =>   '--lock-multiple --compress-algo 1 ' .
 	                      '--cipher-algo cast5 --force-v3-sigs',
